src/main/java/bio/terra/stairway/StepStatus.java

```diff
@@ -1,7 +1,10 @@
 package bio.terra.stairway;
 
 public enum StepStatus {
-    STEP_RESULT_SUCCESS,
-    STEP_RESULT_FAILURE_RETRY,
-    STEP_RESULT_FAILURE_FATAL;
+    STEP_RESULT_SUCCESS,       // step succeeded, ready for next step
+    STEP_RESULT_YIELD,         // step succeeded and yields the thread; for long waits
```

Comment:
This is the return status that requires someone to "manually" resume the flight, right? If so, "YIELD" is misleading to me because in OS-world, that means that a thread moves to the back of the scheduler's priority queue. It's still going to run, just maybe not right away. And if I've understood this correctly, that's not the case here -- someone has to call Stairway.resume to continue running a yielded flight. Maybe "PAUSE" instead?

Reply:
Renamed to WAIT.